src/condor_starter.V6.1/docker_proc.cpp

```diff
@@ -182,9 +182,11 @@ int DockerProc::StartJob() {
 
 	buildExtraVolumes(extras, *machineAd, *JobAd);
 
+	int *affinity_mask = makeCpuAffinityMask(Starter->getMySlotNumber());
```

Comment:
So as I understand, makeCpuAffinityMask will check if ENFORCE_CPU_AFFINITY is set and if so it will set the affinity_mask (it malloc it, so I guess we also need to free it after calling CreateContainer).
So it's the same config_val for both regular execution and docker. I think it's ok, or you think we need a new dedicated config for CPU_AFFINITY_DOCKER ... ?